src/config.js

```diff
@@ -0,0 +1,16 @@
+
+const port = parseInt(process.env.PORT, 10) || 3030;
+const appSid = process.env.APP_SID;
+const accountSid = process.env.ACCOUNT_SID;
+const apiKey = process.env.API_KEY;
+const apiSecret = process.env.API_SECRET;
+const callerId = process.env.CALLER_ID;
+
+if (!appSid || !accountSid || !apiKey || !apiSecret || !callerId) {
+  console.log('Missing environment variables.\n');
+  process.exit();
```

Comment:
I believe that `process.exit` is not actually recommended, instead we can set `process.exitCode = 1` and then make sure we don't queue anything up to the event loop after this. Instead of "throwing" here, I think we should just return something symbolizing missing environment variables.

https://nodejs.org/api/process.html#processexitcode